build.ts

```diff
@@ -306,6 +328,8 @@ function processMap(pmap: PlacementMap, isStatic: boolean): void {
       ui_equip: params ? objGetUiEquipment(params) : null,
       messageid: params ? (params['MessageID'] || null) : null,
       region: pmap.type == 'MainField' ? towerNames[mapTower.getCurrentAreaNum(obj.data.Translate[0], obj.data.Translate[2])] : "",
+      field_area: area >= 0 ? area : "",
```

Comment:
How does this work? Are you allowed to specify an empty string for an integer field?